inference-engine/src/cldnn_engine/cldnn_common_utils.h

```diff
@@ -185,4 +185,11 @@ inline std::vector<uint16_t> ConvertPermuteOrder(const std::vector<uint16_t>& ie
     return cldnn_order;
 }
 
+inline InferenceEngine::Layout InferenceEngineLayoutFromOVLayout(ov::Layout l) {//not finished
+    std::string layout = l.to_string();
+    if (layout == "[N,H,W,C]") return InferenceEngine::Layout::NHWC;
```

Comment:
It is better to not rely on internal representation of 'to_string' (it is more for debugging).
Try using `if (layout == Layout("NHWC")` instead